docs/sources/tempo/_index.md

```diff
@@ -44,7 +44,8 @@ Distributed tracing visualizes the lifecycle of a request as it passes through a
 Tempo is cost-efficient and only requires an object storage to operate.
 Tempo is deeply integrated with Grafana, Mimir, Prometheus, and Loki.
 You can use Tempo with open source tracing protocols, including Jaeger, Zipkin, or OpenTelemetry.
-<p align="center"><img src="getting-started/assets/trace_custom_metrics_dash.png" alt="Trace visualization in Grafana "></p>
+
+{{< figure src="getting-started/assets/trace_custom_metrics_dash.png" alt="Trace visualization in Grafana" class="w-100p" link-class="w-fit mx-auto d-flex flex-direction-column" >}}
```

Comment:
Thank you. We have some legacy HTML image and paragraph tags in the documentation, like the example that you've updated, that are part of the image clean up tech debt.